### PR TITLE
setViewpoertSize: added type parameter according to documentation

### PR DESCRIPTION
**Pull request by pazone**
According to documentation, ```setViewpoertSize``` should have the boolean parameter ```type```. When ```type``` is set to ```false```, ```setViewpoertSize``` will change window(not viewport) size without indentation and retries.
@christian-bromann did you mean this indentation problem?

### CODE REVIEW

File: lib/commands/setViewportSize.js [hunk 1]
```diff
@@ -37,16 +36,19 @@ module.exports = function setViewportSize(size) {
         throw new ErrorHandler.CommandError('number or type of arguments don\'t agree with setViewportSize command');
     }
 
-    return setSize(this);
+    var shouldIndent = typeof(type) === 'undefined' ? true : type;
```
Comment (christian-bromann) [inline lib/commands/setViewportSize.js]: According to our docs the default value of type is false (apply windowHandleSize), so we should switch `true` and `type`.

File: lib/commands/setViewportSize.js [hunk 2]
```diff
@@ -24,11 +24,10 @@
  *
```
Comment (christian-bromann) [inline lib/commands/setViewportSize.js]: what I meant if indentation is here ^^^ could you also fix this (see line 8 - 14)

File: test/spec/desktop/viewportSize.js [hunk 3]
```diff
@@ -21,4 +21,14 @@ describe('setViewportSize/getViewportSize', function() {
         });
     });
 
+    it('should set window size exactly when parameter \'type\' is false', function() {
+        return this.client.setViewportSize({
+            width: 500,
+            height: 500
+        }, false).windowHandleSize().then(function(res) {
```
Comment (christian-bromann) [inline test/spec/desktop/viewportSize.js]: we already have the test above, type should be `true` and we should make sure that the window size was changed not the viewport size

File: lib/commands/setViewportSize.js (hunk 2 above)
Comment (pazone) [inline lib/commands/setViewportSize.js]: ok

File: lib/commands/setViewportSize.js (hunk 1 above)
Comment (pazone) [inline lib/commands/setViewportSize.js]: As I can see in the doc [here](https://github.com/webdriverio/webdriverio/blob/master/lib/commands/setViewportSize.js#L20)
> param {Boolean}  type  set to `false` to change window size, `true` (default) to change viewport size

Looks like the default value of type is true. Should I edit this doc and change behaviour of type parameter?

Comment (christian-bromann) [inline lib/commands/setViewportSize.js]: I would like to have `false/undefined` as default so the default action is `windowHandleSize`. If someone set type to `true` it changes the viewport size

File: test/spec/desktop/viewportSize.js (hunk 3 above)
Comment (christian-bromann) [inline test/spec/desktop/viewportSize.js]: scratch that above .. we should test if it changes the window size if no type parameter was assigned and viewport size if type parameter is true

**Comment by christian-bromann** [comment]
Sorry about the confusion about the default value of type

File: lib/commands/setViewportSize.js (hunk 1 above)
Comment (pazone) [inline lib/commands/setViewportSize.js]: Ok, it's good idea

**Comment by christian-bromann** [comment]
@pazone I've just seen that the default action was to change the viewport size and __not__ to change the window size. I think we should keep it that way. You're right!

File: lib/commands/setViewportSize.js (hunk 1 above)
Comment (christian-bromann) [inline lib/commands/setViewportSize.js]: @pazone please see the last comment (sorry)

**Comment by pazone** [comment]
So, I've done it anyway. The both decisions looks fine. I can remove last commit, if we want to change viewport(not window) size as default.

**Comment by christian-bromann** [comment]
@pazone mhm, what do you thing is best as default value?

**Comment by pazone** [comment]
Actually ```setViewportSize``` is operating with window size and opportunity to change viewport size is looking as an additional feature. But also we have ```windowHandleSize``` to work with browser window. I thing these commands are little bit overloaded by parameters. I hope we will able to find the way to make a little redesign of these commands later. ```setViewportSize``` should change viewport size by default

File: lib/commands/setViewportSize.js [hunk 4]
```diff
@@ -8,12 +8,12 @@
...
-            width: 500,
-            height: 500
-        })
+    width: 500,
+    height: 500
+ })
  .windowHandleSize(function(err, res) {
-            console.log(res.value); // outputs: "{ width: 500, height: 602 }"
-        })
+    console.log(res.value); // outputs: "{ width: 500, height: 602 }"
```
Comment (christian-bromann) [inline lib/commands/setViewportSize.js]: should be `console.log(res.value); // outputs: "{ width: 500, height: 500 }"` then

**Comment by christian-bromann** [comment]
you know what, I will merge this before confuse you even more, I don't know why I was so confused about the default value. Of course: `setViewportSize` should change the viewport per default. Thanks a lot for that fix!

**Comment by christian-bromann** [comment]
Fixed and published with `v3.2.0`

**Pull request merged**